base/linalg/lapack.jl

```diff
@@ -1014,6 +1262,14 @@ for (gglse, elty) in ((:dgglse_, :Float64),
     end
 end
 
+"""
+    gglse!(A, c, B, d) -> (X,res)
+
+Solves the equation `A * x = c` where `x` is subject to the equality
```

Comment:
this is "solve" in a least squares sense, right? https://software.intel.com/en-us/node/521117 (though don't copy any wording directly from the mkl docs unless we check their doc license carefully)